docker/aws-linux/runtests.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+function pr() {
+  echo -e ${GREEN}$1${NC}
+}
+
+function generate_coverage_report() {
+  if ! find . -iname "*.gcda" | grep -q "."; then
+    echo No coverage data found in "$PWD"
+    return 0
+  fi
+
+  if ! command -v lcov || ! command -v genhtml; then
+    echo 'Failed to generate code coverage report (lcov and genhtml commands not found)'
+    return 1
+  fi
+
+  # Convert coverage data from LLVM to lcov format, see http://logan.tw/posts/2015/04/28/check-code-coverage-with-clang-and-lcov/
+  # (With a more recent LLVM, the same can be achieved with `llvm-cov export lcov`)
+  echo '#!/bin/bash' > llvm-gcov.sh
+  echo 'llvm-cov gcov "$@"' >> llvm-gcov.sh
+  chmod +x llvm-gcov.sh
+
+  lcov --capture \
```

Comment:
I didn't find `lcov` installation on `dockerfile`. Is it included in other package that is being installed?